docker/test_env/postgres.go

```diff
@@ -36,17 +37,26 @@ func WithPostgresDbContainerName(name string) PostgresDbOption {
 	}
 }
 
+func WithPostgresImageVersion(version string) PostgresDbOption {
+	return func(c *PostgresDb) {
+		if version != "" {
+			c.ImageVersion = version
+		}
+	}
+}
+
 func NewPostgresDb(networks []string, opts ...PostgresDbOption) *PostgresDb {
 	pg := &PostgresDb{
 		EnvComponent: EnvComponent{
 			ContainerName: fmt.Sprintf("%s-%s", "postgres-db", uuid.NewString()[0:8]),
 			Networks:      networks,
 		},
-		User:     "postgres",
-		Password: "mysecretpassword",
-		DbName:   "testdb",
-		Port:     "5432",
-		l:        log.Logger,
+		User:         "postgres",
+		Password:     "mysecretpassword",
+		DbName:       "testdb",
+		Port:         "5432",
+		ImageVersion: "15.3",
```

Comment:
Let's make it a constant

Reply:
hm, will it help with anything? Btw, if you think this value should be const, other should be as well right?

Reply:
Yeah other should be as well. Just a good to have stuff. Feel free to ignore now. can be a follow up PR later. It just makes it easy to update and easily readable.